packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -924,6 +924,10 @@ typedef void StateSetter(VoidCallback fn);
 ///    [setState] method, which indicates that some of their internal state
 ///    has changed in a way that might impact the user interface in this
 ///    subtree.
+///  * If a hot-reload occurs, such as initiated by the InteliJ IDEA, the 
+///    [State.reassemble] method is called (if overriden). This provides an 
```

Comment:
You can remove the parenthetical (`(if overriden)`), because it's called regardless. :-)
If you don't want to remove it, please change it to `overridden`.